chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java

```diff
@@ -1263,6 +1267,16 @@ public void initializeState() {
         } finally {
             TraceEvent.end("ChromeTabbedActivity.initializeState");
         }
+
+        // This chrome activity will be used only for WebRTCGameStreaming.
```

Comment:
Only for Service offloading? I think this permission can be used for general cases.

Reply:
I'm going to add the condition using the variable in the manifest.